tests/e2e/api/collection.test.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-import { Aptos, AptosConfig, Network } from "../../../src";
+// import { Aptos, AptosConfig, Network } from "../../../src";
```

Comment:
intentional?

Reply:
Yeah, failed the linter because it's unused imports

Reply:
ahhh right i commented out the tests.  didn't have that context in the snippet view.